### ISSUE TITLE
 Cannot achieve the performance given by your homepage.

### ISSUE THREAD

**Issue by lianxi-fan** [BUPTDM/OpenHINE#12]
It seems that using the default parameter configuration cannot achieve the performance given by your homepage (such as **HAN** and **metapath2vec**). Are the default parameters informative?

**Comment by Theheavens** [comment]
The hyperparameter with the best performance is not always same for different models or datasets. Adjust the parameter learning rate(alpha) and epochs.
